opentelemetry/proto/collector/trace/v1/trace_service.proto

```diff
@@ -16,7 +16,7 @@ syntax = "proto3";
 
 package opentelemetry.proto.collector.trace.v1;
 
-import "opentelemetry/proto/trace/v1/trace.proto";
+import "opentelemetry/proto/traces/v1/traces.proto";
 
 option java_multiple_files = true;
 option java_package = "io.opentelemetry.proto.collector.trace.v1";
```

Comment:
Should we not also change this? See #111. If this would become inconsistent, I'm against this PR.

Reply:
I can change that namespace in Java. I cannot change Lin 17 because that will be backwards incompatible

Reply:
Oh right. With that the folder structure does not match the package name anymore. Then I really think we should not rename the folder.